contracts/cw1-subkeys/src/contract.rs

```diff
@@ -329,15 +413,20 @@ mod tests {
         };
         init(deps, env.clone(), init_msg).unwrap();
 
-        // Add subkeys with initial allowances
-        for (spender, expiration) in spenders.iter().zip(expirations) {
+        // Add subkeys with initial allowances and permissions
+        for ((spender, expiration), permissions) in spenders.iter().zip(expirations).zip(permissions) {
```

Comment:
I would not add permissions here. The setup script is already too confusing for me. And this makes it harder to test individual cases. (Just set balance, just set permissions, set both). I guess they are somehow covered here, but it should be more explicit.

I was already a bit unhappy with this giant setup script and adding more makes it super hard to see what is being tested. I would rather just use this for a few of the targetted tests below, and just call `init` and a few Allowance/Permssions settings explicitly for the tests you are writing